src/test/java/jrm/user/UserSignUpTest.java

```diff
@@ -25,10 +25,33 @@
     private UserRepository userRepository;
 
     @Test
-    @Description("200 ок Базовый сценарий регистрации.")
+    @Description("201 ок Базовый сценарий регистрации.")
     void successfulRegistrationTest() {
         UserDto user = UserDto.builder()
-                .name("test123")
+                .name("qwerty")
+                .email("qwerty@mail.ru")
+                .password("qwerty")
+                .username("qwerty")
+                .build();
+
+        ValidatableResponse response = given().contentType(ContentType.JSON)
+                .body(user)
+                .when()
+                .post(userAuthApiService.getUrlSignUp())
+                .then()
+                .log().all();
+
+        assertEquals(201, response.extract().statusCode(), "ответ не 201");
+        User userAfterSelect = userRepository.fetchUserByUsername(user.getUsername());
```

Comment:
"userAfterSelect" is some weird naming for the "dbUser", I think. So it can be "userAfterCreate" or "dbUser" or anything that really describe why this user is different with initial "user".

Reply:
You are right. 'dbUser' would be more suitable!